GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -40,6 +40,7 @@ public static void Register(GVFSPlatform platform)
         public abstract bool IsProcessActive(int processId);
         public abstract void IsServiceInstalledAndRunning(string name, out bool installed, out bool running);
         public abstract string GetNamedPipeName(string enlistmentRoot);
+        public abstract string GetTelemetryNamedPipeName();

```

Comment:
I think this refers to the pathname of the named pipe to the my telemetry service.
We're using a named pipe on Windows, but I think we're going to use sockets or something (TBD) on the Mac.
I wonder if this can be hidden inside the platform code, or rather does it need to be visible in the shared layers?

Reply:
The pipe name (or socket name, config, or whatever) has been moved into the platform code and out of the public API surface.